lib/modules/apostrophe-schemas/index.js

```diff
@@ -1527,6 +1527,16 @@ module.exports = {
     });
 
     self.addFieldType({
+      name: 'object',
+      converters: {
+        form: function(req, data, name, object, field, callback) {
+          object[name] = data[name];
```

Comment:
This needs to sanitize. See the implementation of the `array` field type which is very similar to what you need to do, minus the outer loop.